src/main/resources/db/migration/V1_8__UPDATE_TABLES_FOR_ABSTRACT_ENTITES.sql

```diff
@@ -56,4 +56,28 @@ UPDATE `USER_ACCOUNT`
   SET CREATED_DATE = '2018-01-09', UPDATED_DATE = '2018-01-09'
   WHERE ID > 0;
 
-ALTER TABLE `USER_ACCOUNT` MODIFY UPDATED_DATE datetime not null;
\ No newline at end of file
+ALTER TABLE `USER_ACCOUNT` MODIFY UPDATED_DATE datetime not null;
+
+------------------- ATTENDANCE ----------------
```

Comment:
please move all this section to new sql file, as I many time already told in webinars, each file should be consistent with database, and once it went into master it can't be changed.